docs/README.md

```diff
@@ -87,18 +92,109 @@ The simulation will then try to connect with the ROS2 driver every few seconds.
+
+### Adding the local computer to the virtual controller whitelist
+
+To access the virtual controller with RWS from a networked ROS2 computer, you will need to whitelist the ROS2 computer's IP on the RobotStudio computer. Without this whitelist, you will get an RAPI Unidentified Error when trying to access the controller. You can verify this if you try to navigate to `<ROBOTSTUDIO_COMPUTER_IP>:80` from the ROS2 computer. The default credentials are:
+
+```
+Username: Default User
+Password: robotics
+```
+
+To whitelist the ROS2 computer's IP, on the RobotStudio computer, navigate to:
+
+`C:\Users\<USER>\AppData\Roaming\ABB Industrial IT\Robotics IT\RobVC\`
+
+Look for a file called `vcconf.xml`. Create it if it does not exist.
+
+Paste the following into`vcconf.xml`:
+
+```
+<VCConfiguration>
+<RemoteVCConfiguration PublicationEnabled="true" />
+<hosts>
+<host ip="ROS2_COMPUTER_IP"/>
+</hosts>
+</VCConfiguration>
+```
+
+Change ROS2_COMPUTER_IP to the IP of the ROS2 computer. Up to 10 IP addresses can be whitelisted by adding additional `<host ip="..."/>` lines.
+
+See [this forum post](https://forums.robotstudio.com/discussion/12082/using-robotwebservices-to-access-a-remote-virtual-controller) for more details.
+
+### Configuring SSH port forwarding
+
+If you'd prefer to avoid the [whitelisting solution](#adding-the-local-computer-to-the-virtual-controller-whitelist), you can also use ssh port forwarding.
+
+[Install OpenSSH](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse#install-openssh-using-windows-settings) on the RobotStudio computer.
+
+Start the SSH server by running the following command in PowerShell as an administrator:
+
+```Start-Service sshd```
+
+On the ROS2 computer, set up the SSH port forward by typing the following command into the terminal:
+
+```ssh -L <DESTINATION_PORT>:localhost:<RWS_PORT> <ROBOTSTUDIO_USER>@<ROBOTSTUDIO_IP>```
+
+- Change `<DESTINATION_PORT>` to a port of your choice. Note that `sudo` is required to use ports 0-1023, as they are reserved. You may also need to change the `rws_port` parameter in the [robot description file](../robot_specific_config/abb_irb1200_support/urdf/irb1200.ros2_control.xacro) to match the port you selected. 
+- Change `<RWS_PORT>` to the port used for RWS on the RobotStudio machine, typically 80.
+- Change `<ROBOTSTUDIO_USER>` to the Windows username on the RobotStudio computer. This account must have a password.
+- Change `<ROBOTSTUDIO_IP>` to the IP of the RobotStudio computer.
+
+Enter the Windows login password for the `<ROBOTSTUDIO_USER>` account when prompted.
+
+This will allow you to access RWS on the ROS2 computer on `127.0.0.1:<DESTINATION_PORT>`. You will also need to change the `rws_ip` parameter in the [robot description file](../robot_specific_config/abb_irb1200_support/urdf/irb1200.ros2_control.xacro) to `127.0.0.1`.
```

Comment:
```suggestion
This will allow you to access RWS on the ROS2 computer on `127.0.0.1:<DESTINATION_PORT>`. When launching ROS, use this value for robotstudio_ip: `127.0.0.1`.
```